strum_helpers/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "strum_helpers"
+version = "0.17.1"
+authors = ["Peter Glotfelty <peter.glotfelty@microsoft.com>"]
+license = "MIT"
+
+description = "Helpful macros for working with enums and strings"
+keywords = ["enum", "string", "macros", "proc-macros"]
+categories = ["development-tools::procedural-macro-helpers", "parsing"]
+
+documentation = "https://docs.rs/strum"
+homepage = "https://github.com/Peternator7/strum"
+readme = "../README.md"
+
+[lib]
+name = "strum_helpers"
+
+[dependencies]
+heck = "0.3"
+proc-macro2 = "1.0"
+syn = { version = "1.0", features = ["parsing", "extra-traits"] }
+
+[features]
+verbose-enumstring-name = []
```

Comment:
Do we need these feature flags on the helpers crate? I think they are just needed in the primary "macros" crate.

Reply:
Sorry life's been super busy. I don't think so, it was mostly copy-paste. I'll update the branch & pr here shortly.